src/test/encoding/readble.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
```

Comment:
This doesn't look like a mds- or osd-targetting change. Maybe we have it (+ directly related parts) in a separated commit?

Reply:
thanks, need to delete it from that commit